authentication/templates/registration/base.html

```diff
@@ -0,0 +1,33 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    {% block title %}{% endblock %}
+</head>
+<body>
+    
+    {% block sidebar %}
+    <ul class="sidebar-nav">
+        <li><a href="{% url 'index' %}">Home</a></li>
+        <li><a href="{% url 'book-list' %}">All books</a></li>
+    </ul>
+    
+    <ul class="sidebar-nav">
+    {% if user.is_authenticated %}
+        <li>User: {{ user.first_name }}</li>
+        <li><a href="{% url 'view-loaned' %}">My Borrowed</a></li>
+        <li><a href="{% url 'logout' %}">Logout</a></li>   
+    {% else %}
+        <li><a href="{% url 'login'%}">Login</a></li>   
```

Comment:
Register button should also be provided in the sidebar, just in case the user has not registered.